### ISSUE TITLE
Error when running

### ISSUE THREAD

**Issue by SyntaxViking** [Sequince/Hearthstone#1]
![image](https://user-images.githubusercontent.com/104625604/165924913-cc9bfed7-49ad-4106-b607-3380c47598da.png)

**Comment by Sequince** [comment]
Hi Akaislut,
Yes, my GitHub is mostly just a reference of code I've written for myself.  I'm not sure how much my script would help someone else unless you knew a fair bit about AutoHotKey.  To use it, you'd need to take images yourself, and modify locations in the code depending on your screen resolution.
;GUI script that types out text when called.SeqGUIMain4(SeqText, SeqX, SeqY, seqColour){GUI, SeqShowMainGUI4:destroyGUI, SeqShowMainGUI4: Color, %seqColour%GUI, SeqShowMainGUI4: Font, s16GUI, SeqShowMainGUI4:Font, cWhiteGUI, SeqShowMainGUI4:+AlwaysOnTopGUI, SeqShowMainGUI4:-Caption ;Hide minimize or drag barGUI, SeqShowMainGUI4:add, text,,%SeqText%GUI, SeqShowMainGUI4:+ToolWindow ;Hide from taskbar/alt tabGUI, SeqShowMainGUI4:show, NoActivate X%SeqX% Y%SeqY% ;NoActivate means doesn't steal focus!}

Basically what the full script does is run through the Mercenaries mode of Hearthstone stage 2 Heroic with Emperor Krush, farming coins and hourly reward for 'in game' currency.
